src/db/queries/seasonQueries.ts

```diff
@@ -15,3 +15,16 @@ export const getSeason = async (id: number) => {
     where: eq(seasonsTbl.id, id),
   });
 };
+
+export const getSeasons = async () => {
+  return await readDb.select().from(seasonsTbl);
```

Comment:
Normally we use the syntax `readDb.query.userTbl.findMany()`

Reply:
Hmm - is this to make it easier if we want to include relations? 
I looked at the documentation for selecting all columns in a table, and they provided this method.
https://orm.drizzle.team/docs/select#select-with-all-columns